aspnetcore/log-mon/metrics/built-in/includes/built-in10.md

```diff
@@ -400,4 +400,97 @@ As this metric is tracking the connection duration, and ideally SignalR connecti
 | `signalr.transport` | string | [SignalR transport type](https://github.com/dotnet/aspnetcore/blob/main/src/SignalR/docs/specs/TransportProtocols.md) | `web_sockets`; `long_polling` | Always |
 
 .
+
+## `Microsoft.AspNetCore.Authorization`
+
+The `Microsoft.AspNetCore.Authorization` metrics report information about [Authorization attempts](xref:security/authorization/introduction) in ASP.NET Core apps:
+
+#### Metric: `aspnetcore.authorization.attempts`
+
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `aspnetcore.authorization.attempts` <!--(https://opentelemetry.io/docs/specs/semconv/dotnet/dotnet-aspnetcore-metrics/)--> | Counter | `{request}` | The total number of requests for which authorization was attempted. |
+
+| Attribute  | Type | Description  | Examples  | Presence |
+|---|---|---|---|---|
+| `user.is_authenticated` | Boolean | Match result | `success`; `failure` | Always |
```

Comment:
I think the description for the `user.is_authenticated` attribute may have been a copy/paste error? It could be something like, "Whether the request came from an authenticated user."

Reply:
Oh, and the examples should be "`true`; `false`"